.devcontainer/Dockerfile

```
@@ -1,13 +1,19 @@
 FROM python:3.12-slim-bullseye as builder
 
+
+ARG DOPPLER_TOKEN
+ARG DOPPLER_PROJECT
+ARG DOPPLER_CONFIG
+
+FROM python:3.12-slim-bookworm as builder
```

Hadolint (2.12.0)

[error] FROM aliases (stage names) must be unique

(DL3024)